pkgs/development/libraries/speex/default.nix

```diff
@@ -16,19 +24,20 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev" "doc" ];
 
   nativeBuildInputs = [ autoreconfHook pkg-config ];
-  buildInputs = [ fftw speexdsp ];
+  buildInputs = [ speexdsp ]
+    ++ lib.optionals withFft [ fftw ];
```

Comment:
As is, this is a mass-rebuild and should target the staging branch. But I think if you re-order the buildInputs elements it can eliminate the rebuilds.

Reply:
Fixed